sensorlib/src/collector/SensorDataCollector.py

```diff
@@ -95,12 +95,6 @@ def __collect_raw_point_data(self, grouped_data, raw_sensor_data):
 
             point = CarlaUtils.vector3d_to_numpy(detection.point)
 
-            # CARLA 0.9.10 has a bug where the y-axis value is negated.
-            # This was resolved in a later release, but CARMA currently
-            # uses 0.9.10. Remove this fix when CARMA upgrades to a
-            # newer CARLA version.
-            point[1] *= -1
-
```

Comment:
Please see the description to see why it was removed and similar logic was placed elsewhere